packages/components/src/snackbar/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { flow, noop } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { RawHTML, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '../';
+
+const stopPropagation = ( event ) => event.stopPropagation();
+const NOTICE_TIMEOUT = 10000;
+
+function Snackbar( {
+	className,
+	status,
+	children,
+	actions = [],
+	__unstableHTML,
+	onRemove = noop,
+} ) {
+	useEffect( () => {
+		// This rule doesn't account yet for React Hooks
+		// eslint-disable-next-line @wordpress/react-no-unsafe-timeout
+		const timeoutHandle = setTimeout( () => {
+			onRemove();
+		}, NOTICE_TIMEOUT );
+
+		return () => clearTimeout( timeoutHandle );
+	}, [] );
+
+	const classes = classnames( className, 'components-snackbar', 'is-' + status );
+
+	if ( __unstableHTML ) {
+		children = <RawHTML>{ children }</RawHTML>;
+	}
+
+	return (
+		<div
+			className={ classes }
+			onClick={ onRemove }
+			tabIndex="0"
+			role="button"
+			onKeyPress={ onRemove }
+		>
```

Comment:
I made the notice tabbable so you can focus it and click it to remove it but I was wondering if I should label it or is the content sufficient?

Reply:
![Screen Shot 2019-05-20 at 11 07 34](https://user-images.githubusercontent.com/699132/58010616-fe60b080-7af0-11e9-8dba-e26d6d670edd.png)

This is how VoiceOver deals with it in macOS/Chrome at the moment.

Reply:
I'm thinking it would be clearer if I apply an `aria-label="__( 'Dismiss this notice' )"`? Thoughts?

Reply:
I went with that to match the current Notice component.